packages/daimo-api/src/server/router.ts

```diff
@@ -56,16 +56,21 @@ export function createRouter(
     return opts.next();
   });
 
+  const readyMiddleware = trpcT.middleware(async (opts) => {
+    if (!notifier.isInitialized) {
+      throw new Error("not ready");
+    }
+    return opts.next();
+  });
```

Comment:
just for my understanding, this is functionally equivalent to previous version right? or is there some edge case this changes?

Reply:
this makes it so *all* routes, not just the health route, return this error before the api server is ready

Reply:
right but no other route should be getting hit before the server is ready and health route is "healthy" right? or did you observe behavior/edge case where it can be?